src/app/launcher/step-indicator/step-indicator.component.html

```diff
@@ -2,8 +2,11 @@
   <input class="f8launcher-application-text-field_input"
          name="applicationTitle" placeholder="New Application" type="text"
          [(ngModel)]="launcherComponent.summary.dependencyCheck.projectName"
-         (keyup.enter)="$event.target.blur();">
+         (keyup.enter)="$event.target.blur();" 
+         pattern="[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*"
+         #check="ngModel" minlength="4">
 </div>
+<div [hidden]="!check.hasError('pattern') && !check.hasError('minlength')" class="error-msg">Invalid Application Name</div>
```

Comment:
I believe we should follow Patternfly here and mark the input with the has-error selector. There is an example of this in the Git provider step. Please remove the div tag.

Reply:
Thinking we should also have a tooltip to show what is a valid entry. You can grab the help text shown in the overlay.